### PR TITLE
Add check for length in Queries Validator

### PR DESCRIPTION
**Pull request by vermakhushboo**
There are some scenarios where we need both length and queries validation.

### CODE REVIEW

File: src/Database/Validator/Queries.php [hunk 1]
```diff
@@ -51,6 +57,10 @@ public function isValid($value): bool
             return false;
         }
 
+        if ($this->length && \count($value) > $this->length) {
```
Comment (fogelito) [inline src/Database/Validator/Queries.php:60]: Not sure we need the first since it is constructed with 0?

Comment (fogelito) [reply src/Database/Validator/Queries.php:60]: besides this looks good 
@abnegate Do you want to add something?

**Comment by fogelito** [comment]
Besides small comment looks ok 👍 
@abnegate?

File: src/Database/Validator/Queries.php (hunk 1 above)
Comment (abnegate) [reply src/Database/Validator/Queries.php:60]: @fogelito It looks like we are using 0 to mean unlimited, so when 0, the \count($value) > $this->length won't be checked

**Review by abnegate — APPROVED** [review]

**Pull request merged**